src/services/PostEventProcessor.ts

```diff
@@ -74,19 +78,17 @@ export class PostEventProcessor {
 				this.logger.error({ message: "Missing or invalid value for userDetails.user_id in event payload" }, { messageCode: MessageCodes.MISSING_MANDATORY_FIELDS });
 				throw new AppError(HttpCodesEnum.SERVER_ERROR, "Missing info in sqs event");
 			}
-
 			const userId = userDetails.user_id;
-			// Do not process the event if the event is already processed or flagged for deletion
-			const isFlaggedForDeletionOrEventAlreadyProcessed = await this.iprService.isFlaggedForDeletionOrEventAlreadyProcessed(userId, eventName);
+			//Do not process the event if the event is already processed or flagged for deletion
+			const isFlaggedForDeletionOrEventAlreadyProcessed = await this.iprServiceSession.isFlaggedForDeletionOrEventAlreadyProcessed(userId, eventName);
 			if (isFlaggedForDeletionOrEventAlreadyProcessed) {
 				this.logger.info( { message: "Record flagged for deletion or event already processed, skipping update" });
 				return "Record flagged for deletion or event already processed, skipping update";
 			}
-
 			let updateExpression, expressionAttributeValues: { [key: string]: any }, expiresOn;
 
-			//Set default TTL to 12hrs to expire any records not meant for F2F
```

Comment:
Are we updating the expiry of records in SessionTable to 6hrs as well instead of 12hrs?

Reply:
The SESSION_RETURN_RECORD_TTL_SECS env var is used when the F2F_YOTI_START event is processed so any record in the sessionTable will have the 11 day TTL

Reply:
ahh... ok i get it. we are setting expiry to 12days when we update the record with YOTI_START event. ok. Will resolve my previous comments.